Microsoft.Identity.Web/HttpContextExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace Microsoft.Identity.Web
+{
+    public static class HttpContextExtensions
+    {
+        /// <summary>
+        /// Keep the validated token in associated with the Http request
+        /// </summary>
+        /// <param name="httpContext">Http context</param>
+        /// <param name="token">Token to preserve after the token is validated so that
+        /// it can be used in the actions</param>
+        public static void StoreTokenUsedToCallWebAPI(this HttpContext httpContext, JwtSecurityToken token)
+        {
+            httpContext.Items.Add("token", token);
```

Comment:
>token [](start = 35, length = 5)

"token" is a very generic string and there is a high chance of overwriting a piiece of data that the developer is saving in the HttpContext.Items collection using the key "token". I'd suggest using something hard to guess like "_Msal_cached_access_token_"